packages/sdk/src/index.spec.ts

```diff
@@ -597,5 +611,74 @@ describe("test SDK class", function () {
         );
       });
     });
+    describe("should retry selecting entry node", function () {
+      it("when creating a request and run onCatch", async function () {
```

Comment:
change description to something concerning deadlock since that is what is being tested